tests/php/tests/PUM_DB_Subscribers_Test.php

```
@@ -0,0 +1,802 @@
+		] );
+
+		$this->assertCount( 1, $results );
+	}
+
+	/**
+	 * Test query with numeric search finds matching IDs.
+	 */
+	public function test_query_numeric_search() {
+		$this->db->create_table();
+
+		$id1 = $this->db->insert( [ 'email' => 'num1@example.com', 'popup_id' => 42 ] );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$id1'. (undefined)

(UnusedLocalVariable)